_data/cryptoservices.yml

```diff
@@ -111,10 +111,11 @@ websites:
   url: https://bitit.io/
   img: Bitit.png
   twitter: Bitit_Gift
-  bch: true
+  bch: false
+  bsv: false
   btc: true
   othercrypto: true
-  doc: https://support.bitit.io/hc/en-us/articles/360000566429-How-to-buy-Bitcoin-Cash-BCH-on-Bitit-by-credit-card-cash-and-wire-transfer-
+  text: Bitit used to support BCH, but as of 2019-01-10 stopped supporting it
```

Comment:
due to international use of the site, I think we might want to use fully qualified names of month/dates moving forward

Reply:
@EchterAgo would you mind making this adjustment in the places you used dates in this PR?

Reply:
I used the ISO 8601 date format as it is unambigous.

Reply:
thats fine then, I guess i'm just not as aware of that. Thanks for staying on top of what we should use

Reply:
text block needs to be uneder `exceptions` tag

Reply:
oh, missed that. will fix it